deployment/scripts/fuel-core-ingress-deploy.sh

```diff
@@ -10,11 +10,21 @@ if [ "${k8s_provider}" == "eks" ]; then
     echo "Updating your kube context locally ...."
     aws eks update-kubeconfig --name ${TF_VAR_eks_cluster_name}
     cd ../ingress/${k8s_provider}
+    echo "Deploying production cluster issuer to ${TF_VAR_eks_cluster_name} ...."
```

Comment:
wait.. I thought we were supposed to be removing components unrelated to the fuel-core service?

Reply:
prod-issuer  is related to Ingress- its need to issue a cert to the certificate, since the ingress is deployed via fuel-core still (Not fuel infrastructure), this is ideal place for it

Reply:
sure, but setting up the ELB and cert issuer is an infra level concern, not an application deployment concern.

Reply:
Do we need to make a new PR in fuel-infrastructure to add the cluster issuer back?

Reply:
its included in current infrastructure PR #31 on the board- set it up on create-k8s.sh script

Reply:
To help explain why I don't think this is the ideal place for the issuer, take a look at the faucet ingress app for example: https://github.com/FuelLabs/faucet/blob/master/deployment/scripts/faucet-ingress-deploy.sh

What if we wanted to deploy the faucet service and the fuel-core service to separate clusters? If we had the production cluster issuer in fuel-core ingress, we'd create an arbitrary dependency between facuet and fuel-core, requiring us to deploy an unused fuel-core ingress to the faucet cluster in order to issue certs for the faucet ingress.

Reply:
makes sense- the infrastructure PR should resolve this.